File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/swift/F05.P3.swift
func probe() -> Int32 {
    let xs: [Int32] = [1, 2, 3]
    let k: Int32 = 0
// BEGIN PROBE F05.P3
let neg = { (a: Int32) in k - a }
let ys = xs.map(neg)
return ys[0]
// END PROBE F05.P3
}

print(probe())

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/go/F05.P3.go
package main

import "fmt"

func probe() int32 {
	xs := []int32{1, 2, 3}
	var k int32 = 0
	// BEGIN PROBE F05.P3
	neg := func(a int32) int32 { return k - a }
	var ys []int32
	for _, v := range xs {
		ys = append(ys, neg(v))
	}
	return ys[0]
	// END PROBE F05.P3
}

func main() {
	fmt.Println(probe())
}
